spec/tool/code_interpreter_spec.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
```

Comment:
I think you may need `require "safe_ruby"` but maybe not.

Reply:
Seems to run fine without it.